### ISSUE TITLE
S(Q) models need function descriptions in documentation (Trac #1187)

### ISSUE THREAD

**Issue by butlerpd** [SasView/sasmodels#188]
As noted by smk78 none of the structure factor functions have their mathematical functions described in the documentation.

Migrated from http://trac.sasview.org/ticket/1187
```json
{
    "status": "closed",
    "changetime": "2019-03-27T16:19:54",
    "_ts": "2019-03-27 16:19:54.220861+00:00",
    "description": "As noted by smk78 none of the structure factor functions have their mathematical functions described in the documentation.",
    "reporter": "butler",
    "cc": "",
    "resolution": "wontfix",
    "workpackage": "SasView Documentation",
    "time": "2018-10-03T02:39:39",
    "component": "sasmodels",
    "summary": "S(Q) models need function descriptions in documentation",
    "priority": "critical",
    "keywords": "",
    "milestone": "SasView 4.3.0",
    "owner": "",
    "type": "defect"
}
```

**Comment by butlerpd** [comment]
Trac update at `2019/03/23 14:39:47`: **butler** changed priority from "major" to "critical"

**Comment by smk78** [comment]
Trac update at `2019/03/27 16:12:28`: **smk78** commented:

The current S(q) models are:

* hardsphere
* hayter_msa
* squarewell
* stickyhardsphere

The wording of the documentation for all of these has been reviewed and tweaked. In particular:

* a useful empirical formula for calculating epsilon has been added to hayter_msa
* a legacy png formula was removed from squarewell
* the meaning of epsilon and tau in the stickyhardsphere docs HAVE BEEN REVERSED so that the meaning of epsilon is now consistent across the S(q) models
* mention of Beta(q) has been added to all S(q) descriptions

**Comment by smk78** [comment]
Trac update at `2019/03/27 16:19:38`: **smk78** commented:

Rather like some of the P(q) models noted in SasView/sasview#1266, the equations for the S(q) models are lengthy and complex. Following a discussion at Code Camp IX it was decided to close this ticket as wontfix.

Note that the S(q) model docs may require some additional editing when Beta(q) documentation becomes available (see #173).

**Comment by smk78** [comment]
Trac update at `2019/03/27 16:19:54`: 
* **smk78** changed resolution from "" to "wontfix"
* **smk78** changed status from "new" to "closed"
